guides/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Remove note about nested locale folders requiring configuration as this is now
+    supported by default since PR #41872 which was released as part of Rails 7.0.
+
+    Fixes #53912.
+
+    *Andrew White*
+
```

Comment:
Hi @pixeltrix 👋 
I don't think doc changes require a changelog entry.

Reply:
Yes, I got misled by the few that are but they're more of the 're-wrote the guide' level than this